server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -587,6 +589,21 @@ export default class MakeAReferralController {
         return
       }
       error = form.error
+      draftOasysRiskInformation = form.editedDraftRiskInformation
+    } else {
+      try {
+        draftOasysRiskInformation = await this.interventionsService.getDraftOasysRiskInformation(
+          accessToken,
+          referralId
+        )
+      } catch (e) {
+        const restClientError = e as RestClientError
+        if (restClientError.status === 404) {
```

Comment:
this looks like a guard against the endpoint not existing in the API - is this nessecary so long as we have contract tests?

Reply:
The first time the user accesses the page the service call will always return NOT_FOUND.

When they click 'save and continue' it saves it in the db and when they return (i.e. from check answers) then we'd expect a draft oasys risk information to be returned.

Reply:
👍